default.nix

```diff
@@ -1,4 +1,4 @@
-# https://nixos.wiki/wiki/Flakes#Using_flakes_project_from_a_legacy_Nix
+# https://wiki.nixos.org/wiki/Flakes#Using_flakes_project_from_a_legacy_Nix
```

Comment:
Unless mistaken, there does not seem to be a section "Using flakes project from a legacy Nix" any longer -- the links redirects me to the top of that wiki page. I don't know Nix enough to guess whether this is important or not (regarding the package code that follows here) or even if the comment has any interest being kept (?)

Reply:
This wiki title was changed in May 2023 https://nixos.wiki/index.php?title=Flakes&diff=next&oldid=9498

Merely renamed = it's now https://wiki.nixos.org/wiki/Flakes#Using_flakes_with_stable_Nix would we want to keep it.

Not sure the internal title hash should be kept though ;)

Reply:
As a developer facing comment, yes the new link hash should probably be updated too since it is relevant to the code being commented.